dev/Repeater/ItemsRepeater.h

```diff
@@ -175,4 +175,9 @@ class ItemsRepeater :
     winrt::IElementFactory m_itemTemplate{ nullptr };
     winrt::Layout m_layout{ nullptr };
     winrt::ElementAnimator m_animator{ nullptr };
+
+    // Bug where DataTemplate with no content causes a crash.
+    // See: https://github.com/microsoft/microsoft-ui-xaml/issues/776
+    // Solution: Have reference to DataTemplate and check if content is not null
+    winrt::DataTemplate m_dataTemplateReference{ nullptr };
```

Comment:
I don't think you need to hold onto the data template. Just a boolean should be good. That way you dont have to load the template every measure - instead you can just check the bool. 

bool isItemTemplateEmpty {false;}

Reply:
This is a great idea! I changed that with latest commit.